packages/salesforcedx-vscode-apex/src/views/icons/iconHelpers.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { extensionUris } from '@salesforce/salesforcedx-utils-vscode';
+import { VSCODE_APEX_EXTENSION_NAME } from '../../constants';
+import { iconKey, ICONS } from './IconsEnum';
+
+/**
+ * Get the Uri for an icon located in the resources directory.
+ * @param key A key from the {@link IconsEnum}
+ * @returns The Uri to the icon image.
+ */
+const getIconPath = (key: iconKey) => {
```

Comment:
So we needed a way to get iconPaths.  My first approach here was to initialize with the extensionContext so I could get the extensionPath.  That lead to some ugliness due to the file using this being initialized early in the activation flow.  To avoid race condition games and have to store the extensionContext here I altered this to use the extensionUri to get the base extension location.